lib/tasks/BackbeatTask.js

```diff
@@ -31,33 +31,11 @@ class BackbeatTask {
         const startTime = Date.now();
         const self = this;
 
-        // FIXME workaround for S3C-4457:
```

Comment:
Is this right? is the block really removed?

Reply:
uhh.... well.... that was not supposed to happen. Let me try this again...